packages/moonstone/Panels/Header.js

```diff
@@ -67,7 +68,7 @@ const HeaderBase = kind({
 		]),
 
 		/**
-		 * When `true`, the header content is indented and the header lines are removed.
+		 * Indent its content and removes lines.
```

Comment:
Verb agreement.  Maybe we should also say 'separator lines'?  It's not clear what lines this refers to as they aren't mentioned elsewhere.